README.md

```diff
@@ -130,6 +132,28 @@ $ helm install my-release \
     neotys/nlweb
 ```
 
+## Custom environment variables
+
+`neoload.configuration.backend.others` and `neoload.configuration.backend.others` sections of *values-custom.yaml* allow to define custom environement variables.
```

Comment:
Same backend key is repeated instead of frontend

Reply:
Should be ok.